README.md

```diff
@@ -109,7 +109,7 @@ If you’re not comfortable sharing your question with the list, contact details
 
 ## Contributions
 
```

Comment:
I think we need to rework this section to say the same as suggested in the opening statement above (L8 comment)

Reply:
We can maybe review this part later on?

Reply:
let complete the PR